src/Microsoft.Azure.SignalR/ClientInvocation/CallerClientResultsManager.cs

```diff
@@ -187,7 +189,28 @@ public bool TryGetInvocationReturnType(string invocationId, out Type type)
         public Type GetStreamItemType(string streamId) => throw new NotImplementedException();
     }
 
-    internal record struct PendingInvocation(Type Type, string ConnectionId, string RouterInstanceId, object Tcs, Action<object, CompletionMessage> Complete)
-    {
+    class PendingInvocation {
```

Comment:
1. Any other classes consuming this PendingInvocation? If yes, move to a separate file, if not, move into CallerClientResultManager class and make it private
2. you can simplify the code as below since only `RouterInstanceId` is settable
    ```cs
    record PendingInvocation(Type Type, string ConnectionId, object Tcs, Action<object, CompletionMessage> Complete)
    {
        public string RouterInstanceId {get; set;}
    }
    ```